### ISSUE TITLE
Authentication issue for file Upload

### ISSUE THREAD

**Issue by VivekIshir** [Gullfaxi171/angular-azure-blob-service#12]
@Maritims I am facing Authentication 
403 (Server failed to authenticate the request. Make sure the value of Authorization header is formed correctly including the signature.)

**Comment by remeezp** [comment]
Hi @VivekIshir, have you managed to fix this issue?

**Comment by SaSanju** [comment]
I am getting the same issue. Is it resolved? Please update

**Comment by Gullfaxi171** [comment]
did you generate a sas token and did you configure CORS well on you Storage Account ?

it looks like a configuration issue, more than an issue with the library

**Comment by SaSanju** [comment]
yes, I have configured CORS as per description. generated SAS token. Still, I am getting error. 
![capture](https://user-images.githubusercontent.com/16513758/53566411-14aa4080-3b82-11e9-8ec5-fbb5cafc469e.PNG)
Please update

**Comment by bjwela** [comment]
I got the same error, but it was due to that I had an auth-interceptor that added my apps authorization headers to the request (JWT token bearer in my case). When I removed the app authorization header then this library worked. 

Just in case someone else see the same issue.

**Comment by Narshe1412** [comment]
I had the same issue as @bjwela but I'm using a custom header in my interceptor to bypass authentication for certain calls.
It could be good if this library can be extended so we can add custom headers and/or the option to have an extended HttpModule to handle non authenticated calls.
